### PR TITLE
Improved README

### PR DESCRIPTION
**Pull request by TiagoDanin**
- Improved README
- User Markdown
- Add License in README

### CODE REVIEW

**Comment by TiagoDanin** [comment]
> Failed to replace env in config: ${NPM_TOKEN}

Travis CI without token

File: README.md [hunk 1]
```diff
@@ -15,50 +63,6 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
...
-   
-amd> 
-     grunt core  
-     Please ensure unit tests pass  
-   cd amd  
-    update version in applicationinsights-core-js.nuspec (version number odd for amd)  
-   nuget pack
-   nuget.exe push -Source "ApplicationInsights-Team" -ApiKey VSTS <Nuget Package Name>.nupkg
\ No newline at end of file
+MIT
```
Comment (markwolff) [inline README.md]: Change to...
Please follow the [link](http://go.microsoft.com/fwlink/?LinkID=510709)

File: README.md [hunk 2]
```diff
@@ -1,7 +1,55 @@
-[![Build Status](https://travis-ci.org/Microsoft/applicationinsights-core-js.svg?branch=master)](https://travis-ci.org/Microsoft/applicationinsights-core-js)
-[![Build status](https://dev.azure.com/mseng/AppInsights/_apis/build/status/AppInsights%20-%20DevTools/1DS%20JavaScript%20SDK%20-%20Core)](https://dev.azure.com/mseng/AppInsights/_build/latest?definitionId=7605)
+# Applicationinsights corejs [![Build Status](https://travis-ci.org/Microsoft/applicationinsights-core-js.svg?branch=master)](https://travis-ci.org/Microsoft/applicationinsights-core-js) [![Build status](https://dev.azure.com/mseng/AppInsights/_apis/build/status/AppInsights%20-%20DevTools/1DS%20JavaScript%20SDK%20-%20Core)](https://dev.azure.com/mseng/AppInsights/_build/latest?definitionId=7605)
```
Comment (markwolff) [inline README.md]: Title on separate line from build badges.
Change title to 'Microsoft Application Insights JavaScript SDK - Core'

**Comment by TiagoDanin** [comment]
@markwolff Done!

**Pull request merged**